parser/parser.go

```diff
@@ -106,18 +103,28 @@ func LoadDogFile() (tm types.TaskMap, err error) {
 		}
 
 		if match {
-			dogfiles = append(dogfiles, file)
-		}
-	}
+			var fileData []byte
+			fileData, err = ioutil.ReadFile(file.Name())
+			if err != nil {
+				return
+			}
 
-	for _, dogfile := range dogfiles {
-		var fileData []byte
-		fileData, err = ioutil.ReadFile(dogfile.Name())
-		if err != nil {
-			return
+			ParseDogfile(fileData, tm)
 		}
-		d = append(d, fileData...)
+
+		// if match {
+		// 	dogfiles = append(dogfiles, file)
+		// }
 	}
 
-	return ParseDogfile(d)
+	// for _, dogfile := range dogfiles {
+	// 	var fileData []byte
+	// 	fileData, err = ioutil.ReadFile(dogfile.Name())
+	// 	if err != nil {
+	// 		return
+	// 	}
+	// 	d = append(d, fileData...)
+	// }
```

Comment:
Nope, this should definitely be going away, it's how we used to concatenate files. Let me fix it quickly